content/en/network_performance_monitoring/network_page.md

```diff
@@ -150,6 +150,36 @@ Select any row from the data table to see associated logs, traces, and processes
 
 {{< img src="network_performance_monitoring/network_page/flow_details.png" alt="Aggregate Connection Details"  style="width:80%;">}}
 
+### Rollups
+
+Rolled up traffic (either port or IP) are displayed with `*`. They represent multiple ports or ips being condensed for a given flow.  
+
+##### IP Addresses:
+
+Rolled up when there are connections to multiple public IPs. For example:
+
+- `hostA` -> `hostB`:`443`
+- `hostA` -> `hostC`:`443`
+- `hostA` -> `hostD`:`443`
+
+**Rollup:** `hostA` -> `*`:`443`, assuming that `hostB`, `hostC`, and `hostD` have public IP addresses.
+
+##### Port: 
```

Comment:
If the roll up is done only on source ports (as the example shows) then we can consider making it explicit:
```
#### Source Port
```

Same comment for the `IP Addresses` section